packaging/docker/run.sh

```diff
@@ -46,6 +46,6 @@ if [ -S "${DOCKER_SOCKET}" ] && [ -n "${PGID}" ]; then
 	fi
 fi
 
-exec /usr/sbin/netdata -u "${DOCKER_USR}" -D -s /host -p "${NETDATA_PORT}" "$@"
+exec /usr/sbin/netdata -u "${DOCKER_USR}" -D -s /host -p "${NETDATA_PORT}" -W set web "web files group" root -W set web "web files owner" netdata "$@"
```

Comment:
As above, I think the owner should be root

```suggestion
exec /usr/sbin/netdata -u "${DOCKER_USR}" -D -s /host -p "${NETDATA_PORT}" -W set web "web files group" root -W set web "web files owner" root "$@"
```